django/core/management/commands/shell.py

```diff
@@ -150,6 +151,22 @@ def get_namespace(self, **options):
         if options and options.get("no_imports"):
             return {}
 
+        verbosity = options["verbosity"] if options else 0
+
+        try:
+            apps.check_models_ready()
+        except AppRegistryNotReady:
+            if verbosity > 0:
+                settings_env_var = os.getenv("DJANGO_SETTINGS_MODULE")
+                self.stdout.write(
+                    "Auto-imports are disabled as settings are not configured.\n"
```

Comment:
I would use "automatic imports" here since that's the terminology we have used when the feature was landed (we did use it in on docs anchors/refs, but for user facing text we stuck with "automatic imports"):
```suggestion
                    "Automatic imports are disabled since settings are not configured.\n"
```